subscriber.go

```diff
@@ -196,14 +192,6 @@ func NewSubscriber(host host.Host, ds datastore.Batching, lsys ipld.LinkSystem,
 
 	scopedBlockHookMutex, scopedBlockHook, blockHook := wrapBlockHook()
 
-	if cfg.rateLimiterFor == nil {
-		// Allows all events. No rate limit.
-		limiter := rate.NewLimiter(rate.Inf, 0)
-		cfg.rateLimiterFor = func(publisher peer.ID) *rate.Limiter {
-			return limiter
-		}
-	}
-
```

Comment:
It is now OK to set the `rateLimiterFor` to `nil`.  in this case, any rate-limiting logic is skipped.